server/target/target.go

```diff
@@ -740,6 +741,173 @@ func readPaginatedTargetsFromPrimaryDB(ctx context.Context, env environment.Env,
 	return fetchTargetsFromPrimaryDB(ctx, env, q, repo)
 }
 
+func GetTargetStats(ctx context.Context, env environment.Env, req *trpb.GetTargetStatsRequest) (*trpb.GetTargetStatsResponse, error) {
+	u, err := env.GetAuthenticator().AuthenticatedUser(ctx)
+	if err != nil {
+		return nil, err
+	}
+	if !isReadFromOLAPDBEnabled(env) {
+		return nil, status.UnimplementedError("Target stats requires an OLAP DB.")
+	}
+	sevenDaysAgo := time.Now().Add(-7 * 24 * time.Hour).UnixMicro()
```

Comment:
Sub?

Reply:
unfortunately, `Sub` takes a time and returns the duration between the two times, whereas `Add` returns a new time where the specified duration has been added.  makes total sense, right?